apps/extension/src/shared/components/grpc-endpoint-form/index.tsx

```diff
@@ -98,7 +99,7 @@ export const GrpcEndpointForm = ({
           </div>
 
           <Button variant='gradient' type='submit' disabled={!isSubmitButtonEnabled}>
-            {submitButtonLabel}
+            {isValidationLoading ? <Loader2 className='size-4 animate-spin' /> : submitButtonLabel}
```

Comment:
Could we add text to go along with the loading spinner? Something like: `Validating rpc <spinner>`